### DISCUSSION TITLE
Change the ` keyword

### DISCUSSION THREAD

**Discussion by DRYN07** [lin-ycv/EverythingPowerToys#103] [Q&A] [answered]
Is it possible to change the ` for other character? Can I make this search default in powertoys?

**Comment by lin-ycv** [comment, marked as answer]
This can be set in powertoys settings, change the `direct activation command`

![image](https://github.com/lin-ycv/EverythingPowerToys/assets/60565737/bd646c05-16be-45a2-a054-1d5ceee09924)


I believe this persistent through upgrades.

**Comment by DRYN07** [comment]
Oh, I just realized that plugin settings are integrated into the PT settings after installation... Thank you anyway :)
